src/Analyzers/Core/Analyzers/AbstractBuiltInCodeStyleDiagnosticAnalyzer_Core.cs

```diff
@@ -43,7 +40,9 @@ protected AbstractBuiltInCodeStyleDiagnosticAnalyzer(ImmutableArray<DiagnosticDe
         SupportedDiagnostics = supportedDiagnostics;
 
         Descriptor = SupportedDiagnostics[0];
```

Comment:
>Descriptor = SupportedDiagnostics[0]; [](http://example.com/codeflow?start=8&length=37)

Here's the spot that was just using the first entry in the array to set the descriptor.